packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -345,6 +352,19 @@ abstract class BucketBase extends Resource implements IBucket {
     return rule;
   }
 
+  /**
+   * Sets the passed bucket as destination bucket for this bucket's access logs.
+   *
+   * @param destinationBucket The bucket where the access logs will be stored.
+   * @param logFilePrefix Optional log file prefix.
+   */
+  public enableServerAccessLogs(destinationBucket: IBucket, logFilePrefix?: string): void {
+    (this.node.defaultChild as CfnBucket).loggingConfiguration = {
```

Comment:
move this to `Bucket` and don't use `node.defaultChild` but rather directly reference the resource.